### DISCUSSION TITLE
Calculation of Unbalanced Forces

### DISCUSSION THREAD

**Discussion by Rockambole** [idea-statica/ideastatica-public#453] [General Discussion]
Hello again,

End forces taken from FE analysis are by definition balanced (it's a prerequisite for static equilibrium). However, the "Unbalanced forces" table in IDEAStatica sometime shows non-zero values. So my question is: how does the program calculate these unbalanced forces? How are moments handled when forces are taken at the member ends connected to joint node (i.e. zero arm)?

Keep up the good work!

Thanks!

Regards, O.S.

**Comment by nathanlukeIDEA** [comment]
Hi @Rockambole,

Are you able to provide an example of when the forces are unbalanced? 

Sometimes if this property is set to true it will cause unbalanced forces due to eccentricities for asymetrical sections like a channel. 
![image](https://github.com/idea-statica/ideastatica-public/assets/93578548/4e33055c-0bc6-4a32-a84a-4a903acf8e31)

But there can be other reasons, like how your project deals with eccentricities, also in some instances calculation of torsions etc. Hard to answer without understanding fully how your base application is working.

**Reply by Rockambole** [reply]
Hi Nathan,

Please see attached files related to my test project...

In my model, there are no eccentricities (i.e. "work point" for all incoming members are the connection node).
I have followed example at https://github.com/idea-statica/ideastatica-public/wiki/IOM-Member-Force-Result-Data and the status of my project is as follows:
1. I transfer forces at incoming member ends connected to the node. For some members, position will be 1.0 and some 0.0 (depending on node incidences
2. All load values (for all dofs) are transferred correctly (both sign and magnitude). This has been thoroughly checked.
3. In the example, there is this: resLc.Loading.Items.Add(new ResultOfLoadingItem() { Coefficient = 1.0 });
    -> What is the significance of "Coefficient"?
4. Also in the example:
        ResultOnSection resSec = new ResultOnSection();
	resSec.AbsoluteRelative = AbsoluteRelative.Relative; 
        resSec.Position = value // where value = 0.0 at member start and 1.0 at member end
In fact, this Position parameter doesn't seem to have any effect (I have tried both values for all + "correct" values for all. No change in "Unbalanced force" table...

I feel we are close...

Please advise.

Thank you, O.S.
![image](https://github.com/idea-statica/ideastatica-public/assets/16279406/76856745-51fb-4bf3-9ee1-89a66f95ecb4)
 
PS. Seems like I can't attach xml or IDEACon files here. I'll send by separate mail:)
